packages/next-source-maps/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@zeit/next-source-maps",
```

Comment:
Should it be called `@zeit/next-production-source-maps` 🤔

Reply:
Nope